src/utils/run.ts

```diff
@@ -92,6 +94,7 @@ export async function run(opts: RunOpts): Promise<Deno.Process> {
 
   const proc = Deno.run({
     cmd: [Deno.execPath(), "eval", ...args, runner],
+    env: opts.env,
```

Comment:
I think it's better to merge `.env` values and actual env variables which the process actually has. Otherwise the command like below doesn't work and it could be confusing:
```bash
SECRET=foobar deployctl run main.ts
```

So I'd suggest something like the below:

```suggestion
    env: { ...opts.env, ...Deno.env.toObject() }
```

Reply:
Deno seems already merging them...